R/vdiagram-data.R

```diff
@@ -1,33 +1,45 @@
+#' Parse Vdiagram Data
+#' @description Takes in a V diagram report and returns the necessary data for a V Diagram
+#' @param reportObject An R object with the raw data required for a V Diagram
+#' @return A list containing the relevant data for a V Diagram
+#'
 parseVDiagramData <- function(reportObject){
   ratingShifts <- fetchRatingShifts(reportObject)
   
-  shiftPoints <- ratingShifts$shiftPoints
+  ### Which of these are required? None of them stop if they're not valid currently.
+  ### Alternatively, do we want to assign validParam(blah, blah) to each (i.e. shiftPoints <- validParam(etc,etc))
+  ### I assume we do as otherwise the "validParam" statements do nothing.
```

Comment:
Agree on asking Laura to say which are necessary for the plot. I personally like the two line option because it makes it more evident that it's a two step thing, but don't have strong opinion about it.